code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -1140,41 +1141,67 @@
 
 /datum/ammo/xeno/New()
 	..()
-	accuracy = config.med_hit_accuracy
+	accuracy = config.max_hit_accuracy
+	accurate_range = config.short_shell_range
+	shell_speed = config.reg_shell_speed
+	max_range = config.short_shell_range
 	accuracy_var_low = config.low_proj_variance
 	accuracy_var_high = config.low_proj_variance
-	max_range = config.short_shell_range
 
 /datum/ammo/xeno/toxin
 	name = "neurotoxic spit"
-	damage_falloff = 0
-	debilitate = list(1,2,0,0,0,0,0,0)
+	ammo_reagents = list("xeno_toxin" = 5)
+	debilitate = list(0.5,0.5,0,0,0,0,0,0)
 	flags_ammo_behavior = AMMO_XENO_TOX|AMMO_IGNORE_RESIST
 	spit_cost = 50
 
-/datum/ammo/xeno/toxin/New()
+/datum/ammo/xeno/toxin/on_hit_mob(mob/M, obj/item/projectile/P)
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if(C.status_flags & XENO_HOST && istype(C.buckled, /obj/structure/bed/nest) || C.stat == DEAD)
+			return
 	..()
-	shell_speed = config.reg_shell_speed
-	max_range = config.close_shell_range
+
+/datum/ammo/xeno/toxin/upgrade1
+	name = "neurotoxic spit"
+	ammo_reagents = list("xeno_toxin" = 5.75)
+
+/datum/ammo/xeno/toxin/upgrade2
```

Comment:
Why different ammo types if you've added a `reagent_multiplier` to `generate_bullet` ?